### ISSUE TITLE
case sensitivity of mqtt topics

### ISSUE THREAD

**Issue by freol35241** [ThomDietrich/miflora-mqtt-daemon#81]
Hi, I man having some trouble with mqtt topics being case sensitive. 
https://github.com/ThomDietrich/miflora-mqtt-daemon/blob/master/miflora-mqtt-daemon.py#L141

This line makes the base topic all lower case. Any particular reason for this?

**Comment by ThomDietrich** [comment]
Not at all. MQTT is case-sensitive and so should the daemon be. Would you like to propose a PR?
